### DISCUSSION TITLE
How to change Logo

### DISCUSSION THREAD

**Discussion by gfo2007** [serenity-is/Serenity#6572] [Help]
Hi,
Previously, suggestions were made on how to change the site logo at wwwroot/Content/Site/site.theme.less

I can't find this file in my serenity project. Does anyone know where I can change the logo of my application?

Regards?

**Comment by edwardch** [comment]
You can do it for the sidebar here: https://github.com/serenity-is/Serene/blob/ec582936835f1f0d19057c535222a93051f6d4de/Serene/Serene.Web/Views/Shared/_Sidebar.cshtml#L59
